BackEnd/src/main/kotlin/cub/trackmyoffer/FeatureProviderRouting.kt

```diff
@@ -31,6 +75,373 @@ fun Route.featureProviderRouting(httpClient: HttpClient, config: FeatureProvider
                 val response = httpClient.get("${config.remote}/")
                 call.respondText(response.bodyAsText(), status = response.status)
             }
+
+            post("/profile") {
+                val profileReq = call.receive<ProfileData>()
+
+                profileReq.id = extractUserId(call)
+
+                val remoteResponse: HttpResponse = httpClient.post("${config.remote}/api/profile") {
+                    contentType(ContentType.Application.Json)
+                    setBody(profileReq)
+                }
+
+                val text = remoteResponse.bodyAsText()
+                call.respond(
+                    status = remoteResponse.status,
+                    message = text
+                )
+            }
```

Comment:
Generally, if those handlers are ready, I'd suggest leaving them under v0 as is (with minor changes maybe), and if we need to make more substantial changes, make a new version of api -- so that there always is *something* we can fall back to.

And to do so, it may be reasonable to move FP's endpoints under v0 too